diff --git a/po/Makefile b/po/Makefile
index 3e75acf..3fd2355 100644
--- a/po/Makefile
+++ b/po/Makefile
@@ -75,25 +75,7 @@ DISTFILES = $(DISTFILES.common) Makevars POTFILES.in \
 $(POFILES) $(GMOFILES) \
 $(DISTFILES.extra1) $(DISTFILES.extra2) $(DISTFILES.extra3)
 
-POTFILES = \
-     ../ext/gsettings/org.freedesktop.gstreamer.default-elements.gschema.xml.in \
-     ../ext/resindvd/resindvdsrc.c \
-     ../ext/smoothstreaming/gstmssdemux.c \
-     ../ext/sndfile/gstsfsink.c \
-     ../ext/sndfile/gstsfsrc.c \
-     ../ext/sndio/sndiosink.c \
-     ../ext/sndio/sndiosrc.c \
-     ../gst/aiff/aiffparse.c \
-     ../gst/asfmux/gstasfmux.c \
-     ../gst/camerabin2/camerabingeneral.c \
-     ../gst/camerabin2/gstcamerabin2.c \
-     ../gst/camerabin2/gstviewfinderbin.c \
-     ../gst/dvdspu/gstdvdspu.c \
-     ../gst/mpegtsdemux/mpegtsbase.c \
-     ../gst/nuvdemux/gstnuvdemux.c \
-     ../gst/videomeasure/gstvideomeasure_collector.c \
-     ../sys/dvb/gstdvbsrc.c \
-     ../sys/dvb/parsechannels.c
+POTFILES = 
 
 CATALOGS =  af.gmo az.gmo bg.gmo ca.gmo cs.gmo da.gmo de.gmo el.gmo en_GB.gmo eo.gmo es.gmo eu.gmo fi.gmo fr.gmo gl.gmo hr.gmo hu.gmo id.gmo it.gmo ja.gmo ky.gmo lt.gmo lv.gmo mt.gmo nb.gmo nl.gmo or.gmo pl.gmo pt_BR.gmo ro.gmo ru.gmo sk.gmo sl.gmo sq.gmo sr.gmo sv.gmo tr.gmo uk.gmo vi.gmo zh_CN.gmo
 
diff --git a/po/POTFILES.in b/po/POTFILES.in
index 0547492..e69de29 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -1,18 +0,0 @@
-ext/gsettings/org.freedesktop.gstreamer.default-elements.gschema.xml.in
-ext/resindvd/resindvdsrc.c
-ext/smoothstreaming/gstmssdemux.c
-ext/sndfile/gstsfsink.c
-ext/sndfile/gstsfsrc.c
-ext/sndio/sndiosink.c
-ext/sndio/sndiosrc.c
-gst/aiff/aiffparse.c
-gst/asfmux/gstasfmux.c
-gst/camerabin2/camerabingeneral.c
-gst/camerabin2/gstcamerabin2.c
-gst/camerabin2/gstviewfinderbin.c
-gst/dvdspu/gstdvdspu.c
-gst/mpegtsdemux/mpegtsbase.c
-gst/nuvdemux/gstnuvdemux.c
-gst/videomeasure/gstvideomeasure_collector.c
-sys/dvb/gstdvbsrc.c
-sys/dvb/parsechannels.c
